student.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'person'
+
+# for students
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] Kindly let's set a getter method here `attr_reader :classroom` as we are only supposed to read the properties.